### DISCUSSION TITLE
I'm a Ui/Ux Designer

### DISCUSSION THREAD

**Discussion by ronanoor** [WeAreMahsaAmini/Core#104] [Introductions]
I'm a Ui/Ux Designer and junior front-end Developer

**Comment by sinahnyazdi** [comment]
Hi Romina, great to have you here. can you help with this? https://github.com/WeAreMahsaAmini/Core/discussions/124
I think we need to use the main slogen (زن زندگی آزادی) in both Farsi and English or better ideas
